commons/src/main/res/values-da/strings.xml

```diff
@@ -354,11 +354,11 @@
     <!-- New app (do not translate anything on the 4th line) -->
     <string name="new_app">
         <![CDATA[
-            Hej<br><br>
-            - synes lige du skulle vide at der er kommet en ny app:<br><br>
+            Hey,<br><br>
+            just letting you know that a new app has been released recently:<br><br>
             <a href="%1$s">%2$s</a><br><br>
-            Du kan downloade den med et klik på titlen.<br><br>
-            Tak
+            You can download it by pressing the title.<br><br>
+            Thanks
```

Comment:
also not overriding these

Reply:
Not?

Reply:
no :-)
I must have made suggestions to the file too early after the previous suggestions.